lib/smart_answer_flows/student-finance-calculator.rb

```diff
@@ -6,12 +6,19 @@ def define
       status :published
       satisfies_need "100133"
 
+      calculator = Calculators::StudentFinanceCalculator.new
+
       #Q1
       multiple_choice :when_does_your_course_start? do
         option :"2016-2017"
 
+        calculate :calculator do
+          calculator
```

Comment:
Yeah, I understand your suggestion.. The scope for `calculator` will break across the flow